eng/build.ps1

```diff
@@ -118,18 +118,30 @@ if ($vs) {
 
   if (-Not (Test-Path $vs)) {
     $solution = $vs
-    # Search for the solution in libraries
-    $vs = Split-Path $PSScriptRoot -Parent | Join-Path -ChildPath "src\libraries" | Join-Path -ChildPath $vs | Join-Path -ChildPath "$vs.sln"
+
+    # Search for the solution in coreclr
+    $vs = Split-Path $PSScriptRoot -Parent | Join-Path -ChildPath "src\coreclr\src" | Join-Path -ChildPath $vs | Join-Path -ChildPath "$vs.sln"
```

Comment:
What about mono's corelib?

Reply:
We control that via the RuntimeFlavor switch.

Reply:
Where does that solution live? I can add it...

Reply:
ah, I found it `src\mono\netcore`

Reply:
@ViktorHofer, do we have an existing mechanism to check if any property is `RuntimeFlavor=Mono`, or do I effectively need too do something like `if ($properties.Where({$_ -eq "/p:RuntimeFlavor=Mono"}, 'First').Count -gt 0`?

Reply:
You would want to add the RuntimeFlavor as a powershell param at the root of the build.ps1 script and then condition on it. We haven't yet needed the differentiation so you are the first one who stumbles upon this.

Reply:
Should be done. I also added it to `build.sh` for parity.